src/main/java/io/cryostat/credentials/Credentials.java

```diff
@@ -89,7 +111,9 @@ public RestResponse<Void> create(
         credential.username = username;
         credential.password = password;
         credential.persist();
-        return ResponseBuilder.<Void>created(URI.create("/api/v2.2/credentials/" + credential.id))
+        return ResponseBuilder.<Credential>created(
+                        uriInfo.getAbsolutePathBuilder().path(Long.toString(credential.id)).build())
```

Comment:
@aali309 see this as well - I found this nice way to set the Location header in responses relative to the current request path. This pattern will apply most of the time to `@POST` endpoints, and it saves us having to update so many hardcoded strings in the future.